sandbox.rmd

```diff
@@ -384,3 +384,56 @@ sim1sbind <- function(x, bmax, kd, cv, reps, log=F) {
 sim1sbind(x, bmax, kd, 0.1, 5, log = F)
 ```
 
+Let's update this:
+
+right now, the stat_function call is just plotting an exact line for the input values of bmax and kd. It is not responding to the variability of the chosen points. 
```

Comment:
maybe it would be better to have two functions? one for linear x entry, another for log x entry. That way you wouldn't need stat_function, and could draw the line with geom_smooth

Reply:
See my below comment

Also, all final changes I put in the <function_name>.R. You can basically ignore sandbox.rmd as far as commentary goes. It just shows the successive iterations until I get to the final function which I take into the .R file